src/js/hca-rjsf/config/form.js

```diff
@@ -265,7 +296,199 @@ const formConfig = {
               isVaServiceConnected,
               receivesVaPension
             }
+          }
+        }
+      }
+    },
+    householdInformation: {
+      title: 'Household Information',
+      pages: {
+        financialDisclosure: {
+          path: 'household-information/financial-disclosure',
+          title: 'Financial disclosure',
+          uiSchema: {
+            'ui:title': 'Financial disclosure',
+            'ui:description': financialDisclosureText,
+            discloseFinancialInformation: {
+              'ui:title': 'Do you want to provide your financial information?',
+              'ui:widget': 'yesNo'
+            }
           },
+          schema: {
+            type: 'object',
+            required: ['discloseFinancialInformation'],
+            properties: {
+              discloseFinancialInformation
+            }
+          }
+        },
+        spouseInformation: {
+          path: 'household-information/spouse-information',
+          title: 'Spouse’s information',
+          // TODO: When veteranInformation is completed, uncomment the maritalStatus comparison
+          depends: (data) => {
+            return data.discloseFinancialInformation; // &&
+              // data.veteranInformation.data.maritalStatus === 'married'
+          },
+          uiSchema: {
+            'ui:title': 'Spouse’s information',
+            'ui:description': 'Please fill this out to the best of your knowledge. The more accurate your responses, the faster we can process your application.',
+            spouseFullName: _.merge(fullNameUI, {
+              'ui:title': 'Spouse Name'
+            }),
+            spouseSocialSecurityNumber: _.merge(ssnUI, {
+              'ui:title': 'Spouse’s social security number',
+            }),
+            spouseDateOfBirth: {
+              'ui:title': 'Date of birth'
+            },
+            dateOfMarriage: {
+              'ui:title': 'Date of marriage'
+            },
+            sameAddress: {
+              'ui:title': 'Do you have the same address as your spouse?',
+              'ui:widget': 'yesNo'
+            },
+            cohabitedLastYear: {
+              'ui:title': 'Did your spouse live with you last year?',
+              'ui:widget': 'yesNo'
+            },
+            provideSupportLastYear: {
+              'ui:title': 'If your spouse did not live with you last year, did you provide financial support?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                // Only show if 'No' is selected for cohabitedLastYear
+                hideIf: (formData) => formData.cohabitedLastYear !== false
```

Comment:
Yeah, that's exactly it. I need it to expand only if "No" is selected, and as far as I know, `expandUnder` will only work if "Yes" is selected.